frontend/src/metabase/css/core/transitions.module.css

```diff
@@ -1,10 +1,18 @@
+:root {
```

Comment:
I believe this will leak outside of the sdk, as we import all our css modules via `frontend/src/metabase/css/core/index.css` which is used in the sdk.
Can we scope this down to the class introduced with https://github.com/metabase/metabase/pull/50466 ?

Reply:
Or we can prefix it with `--mb` which is what we do for colors

Reply:
Thank you, fixed